src/main/java/edu/kis/powp/jobs2d/events/SelectBackLogListOptionListener.java

```diff
@@ -0,0 +1,28 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.gui.backlog.BackLogItem;
+import edu.kis.powp.jobs2d.command.gui.backlog.BackLogList;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+
+import javax.swing.*;
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.ListSelectionListener;
+
+public class SelectBackLogListOptionListener implements ListSelectionListener {
+
+	private DriverCommandManager commandManager;
+
+	public SelectBackLogListOptionListener(DriverCommandManager commandManager) {
+		this.commandManager = commandManager;
+	}
+
+	@Override
+	public void valueChanged(ListSelectionEvent e) {
+		if (e.getValueIsAdjusting() == false) {
+			BackLogItem selectedValue = ((JList<BackLogItem>)e.getSource()).getSelectedValue();
```

Comment:
It would be best to make this listener independent from GUI. 
Could ListSelectionEvent with ListModel cooperate to get the item from history?